sdks/java/core/src/main/java/org/apache/beam/sdk/values/WindowedValue.java

```diff
@@ -49,6 +50,9 @@ public interface WindowedValue<T> {
   @Nullable
   String getRecordId();
 
+  @Nullable
+  Context getContext();
```

Comment:
```suggestion
  Context getOpenTelemetryContext();
```
@kennknowles same here, right?